### PR TITLE
Update link for wechaty-puppet-padchat

### PR DESCRIPTION
**Pull request by LinuxSuRen**
## I'm submitting a...

- [ ] Bug Fix
- [ ] Feature
- [x] Other (Refactoring, Added tests, Documentation, ...)

## Checklist

- [ ] Commit Messages follow the [Conventional Commits](https://conventionalcommits.org/) pattern
  - A feature commit message is prefixed "feat:"
  - A bugfix commit message is prefixed "fix:"
- [ ] Tests for the changes have been added

## Description

Correct a stale link.
[wechaty-puppet-padchat](https://github.com/lijiarui/wechaty-puppet-padchat) is no longer maintain. The new one is [wechaty-puppet-padpro](https://github.com/botorange/wechaty-puppet-padpro).

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,7 +157,7 @@ A `Bot` is a Wechaty instance that control a specific [wechaty-puppet](https://g
     2. `optoins.puppet?: string` select which puppet provider we want to use. must be one of the:
         1. [wechaty-puppet-puppeteer](https://github.com/chatie/wechaty-puppet-puppeteer) - Angular Hook for Web Wechat <- This is the DEFAULT
         2. [wechaty-puppet-wechat4u](https://github.com/chatie/wechaty-puppet-wechat4u) - HTTP API for Web Wechat
-        3. [wechaty-puppet-padchat](https://github.com/lijiarui/wechaty-puppet-padchat) - iPad App Protocol
+        3. [wechaty-puppet-padchat](https://github.com/botorange/wechaty-puppet-padpro) - iPad App Protocol
```
Comment (huan) [inline README.md]: Please change the link text from `padchat` to `padpro` as well.

**Comment by huan** [comment]
It's great to see your update.

We will be able to merge this PR after you signed the CLA. Please click the ` CLA not signed yet` button to sign it, thanks!

**Comment by LinuxSuRen** [comment]
Sorry about that. I don't know why I just ignore that.

**Comment by huan** [comment]
Merged.

Thank you so much for the fix!

And please contact @lijiarui if you would like to join our contributor wechat group: you will be welcome to join if you like. :smile:

**Comment by LinuxSuRen** [comment]
@huan Thanks for the invitation. @lijiarui Please help me to join the group.

**Comment by lijiarui** [comment]
@LinuxSuRen  Please add my wechat: 13141321843 Thanks!

**Comment by LinuxSuRen** [comment]
Copy that.@lijiarui

**Comment by huan** [comment]
@LinuxSuRen  CQ CQ DE BG1FMO K.

**Comment by huan** [comment]
@LinuxSuRen Could you also fix this part?

<https://github.com/Chatie/wechaty/blob/master/README.md#3-switch-protocolpuppet>

**Comment by LinuxSuRen** [comment]
@huan I don't too much about this. Just try to update it. #1838

**Comment by lijiarui** [comment]
@LinuxSuRen Could you add my wechat please? And please left a comment when you add my wechat, or I may miss your friend request! Thanks!

**Comment by LinuxSuRen** [comment]
@lijiarui Copy that.

**Pull request merged**